### PR TITLE
Added examples and definition - CSS

### PR DESCRIPTION
**Pull request by chef-danny-d**
- added two examples with the most used values
- added definition

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by b3u** [comment]
LGTM
BTW, use one backtick instead of three for inline code. "(```initial``` vs. `initial`)

**Comment by chef-danny-d** [comment]
@b3u appreceat it!!! changed it to single ticks.

File: guide/english/css/graceful-degradation/index.md [hunk 1]
```diff
@@ -17,4 +17,18 @@ title: Graceful Degradation
...
+
+[How can polyfills be used as a fallback for older browsers](https://developer.mozilla.org/en-US/docs/Glossary/Polyfill)
+
+[What is polyfill](https://remysharp.com/2010/10/08/what-is-a-polyfill)
+
+[Online HTML beautifier](https://www.10bestdesign.com/dirtymarkup/)
+
+[W3C CSS validator](http://jigsaw.w3.org/css-validator/)
+
+[Online CSS linter](http://csslint.net/)
```
Comment (huyenltnguyen) [inline guide/english/css/graceful-degradation/index.md]: This website is not supporting `https`, which is not conform with the [freeCodeCamp's Style Guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#links). Please remove it from your changes.

**Review by huyenltnguyen — CHANGES_REQUESTED** [review]

**Review by huyenltnguyen — APPROVED** [review]

**Comment by huyenltnguyen** [comment]
Hi @chef-danny-d, thank you for the PR!

I really appreciate your effort to improve the article. I'm merging your PR now.

Congratulations on your first contribution to this repo ✨ 🎉 We look forward to your next.

Happy coding!

**Pull request merged**